packages/onchainkit/package.json

```diff
@@ -40,6 +40,7 @@
     "graphql-request": "^6.1.0",
     "qrcode": "^1.5.4",
     "tailwind-merge": "^2.3.0",
+    "usehooks-ts": "^3.1.1",
```

Comment:
The current implementation of `useIsMounted` in `src/internal/hooks/useIsMounted` doesn't let you check if the component is mounted at the end of the component lifecycle.

Rather than update the hook and its usage internally, I figured bringing in this library would be better since it also covers a _lot_ of the hooks we have in our codebase, and it's been extremely battle tested.